.github/workflows/php.yml

```
@@ -21,16 +21,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2.0.0
+        uses: actions/checkout@v3.0.0
```

I think we may use v3

```suggestion
        uses: actions/checkout@v3
```